File: 백준/Silver/10773. 제로/제로.js
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().split('\n');
var K = parseInt(input[0]);

let stack = [];

for (let i = 1 ; i <= K ; i++) {
    
    let n = parseInt(input[i]);
    
    if ( n == 0 ) {
        stack.pop();
    } else 
        stack.push(n);
}

let sum = 0;

for (const n of stack) {
    sum += n;
}

console.log(sum);

File: 백준/Silver/10773. 제로/제로.py
import sys
input = sys.stdin.readline

K = int(input())

stack = []

for _ in range(K):
    val = int(input())
    if val == 0:
        stack.pop()
    else:
        stack.append(val)

sum = 0

for num in stack:
    sum += int(num)

print(sum)